pulp_container/app/access_policy.py

```diff
@@ -17,8 +17,8 @@ def has_permissions(namespace, user, permission):
         try:
             namespace = models.ContainerNamespace.objects.get(name=namespace)
         except models.ContainerNamespace.DoesNotExist:
-            # check model permissions for namespace creation
-            return user.has_perm("container.add_containernamespace")
+            # Anyone can create a new namespace
+            return True
```

Comment:
Na, we don't want that. If anyone should be able to create a new namespace, than the access_policy needs to reflect that.

Reply:
Yeah, I'd appreciate some background behind the decision of allowing creation of namespaces by anyone.